x/millions/keeper/keeper_pool.go

```diff
@@ -435,6 +435,24 @@ func (k Keeper) UpdatePool(
 	return nil
 }
 
+func (k Keeper) KillPool(ctx sdk.Context, poolID uint64) (types.Pool, error) {
```

Comment:
Please rename this method as `UnsafeKillPool` and add appropriate comment to mention the temporary and exceptional use of this feature.